frontend/src/components/orgFacilities/ManageFacilities.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <v-container class="pa-0 text-body-1" fluid>
+    <v-row no-gutters>
+      <v-col>
+        <p>View and update your facilities.</p>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col>
+        <h2>Facility Details</h2>
+        <p class="mt-2">
+          <v-icon class="noticeInfoIcon">mdi-information</v-icon>
+          Note: The information marked with * will be shared on Child Care Map. Please do not include any personal
+          information.
+        </p>
+      </v-col>
+    </v-row>
```

Comment:
I think this information will be moving to the details screen.

Reply:
Removed